code/modules/power/lighting.dm

```diff
@@ -476,12 +470,15 @@
 	var/obj/item/weapon/light/L = new light_type()
 	L.status = status
 	L.rigged = rigged
-	L.brightness = brightness
 
 	// light item inherits the switchcount, then zero it
 	L.switchcount = switchcount
 	switchcount = 0
 
+	L.l_range = l_range
```

Comment:
If you var edit the range of the light fitting, and then remove the bulb, the bulb inherits the light fitting's var editing